src/main.js

```diff
@@ -151,29 +157,39 @@ function areInputsDefined(userInputs,errorMsgIndex) {
 }
 
 function displayTimeSection() {
-    document.getElementById('user-accomplish').innerText = currentActivity.description;
-    currentActivity.minutes < 10 ? displayMin.innerText = `0${currentActivity.minutes}` : displayMin.innerText = currentActivity.minutes;
-    currentActivity.seconds < 10 ? displaySec.innerText = `0${currentActivity.seconds}` : displaySec.innerText = currentActivity.seconds;
+  document.getElementById('user-accomplish').innerText = currentActivity.description;
+  currentActivity.minutes < 10 ? displayMin.innerText = `0${currentActivity.minutes}` : displayMin.innerText = currentActivity.minutes;
```

Comment:
This could be refactored to make the assignment once for each ternary. i.e. 
```
something = condition ? first : second

vs. 

condition ? something = first : something = second
```